src/textarea/src/Textarea.js

```diff
@@ -116,6 +123,9 @@ class Textarea extends PureComponent {
         {...(disabled ? { color: 'muted' } : {})}
         {...Textarea.styles}
         {...props}
+        style={{
+          resize: resize      
+        }}
```

Comment:
Ideally this should be handled by `ui-box` not Evergreen. That way it's handled anytime you use a `textarea` not just Evergreen's `<Textarea />` component.

Reply:
As far as I could tell, ui-box doesn't include a resize prop

Reply:
You are correct! I'm suggesting we add it there.

Reply:
Ya for sure, that makes sense